classes/ObjectModel.php

```diff
@@ -891,19 +891,25 @@ public function delete()
 
         $this->clearCache();
         $result = true;
+
         // Remove association to multishop table
-        if (Shop::isTableAssociated($this->def['table'])) {
+        if (count($this->id_shop_list)) {
+            $id_shop_list = $this->id_shop_list;
```

Comment:
Kind of question or suggestion.
Should we avoid mixing the cases in this class or should we make the new code correct (with the camelCase)? I think whenever we add a new code in legacy classes, we use vars in camelCase everywhere else no?
```suggestion
            $shopIdsList = $this->id_shop_list;
```

Reply:
Yep you're right I'll change this since it's a new variable anyway